sw/host/tests/chip/spi_device_smoketest/src/main.rs

```diff
@@ -0,0 +1,136 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, Result};
+use clap::Parser;
+use regex::Regex;
+use std::fs;
+use std::path::PathBuf;
+use std::time::Duration;
+
+use object::{Object, ObjectSection, ObjectSymbol};
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::execute_test;
+use opentitanlib::io::spi::Transfer;
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::uart::console::{ExitStatus, UartConsole};
+
+#[derive(Debug, Parser)]
+struct Opts {
+    #[command(flatten)]
+    init: InitializeTest,
+
+    /// Console receive timeout.
+    #[arg(long, value_parser = humantime::parse_duration, default_value = "600s")]
+    timeout: Duration,
+
+    /// Name of the SPI interface to connect to the OTTF console.
+    #[arg(long, default_value = "BOOTSTRAP")]
+    spi: String,
+
+    /// Path to the firmware's ELF file, for querying symbol addresses.
+    #[arg(value_name = "FIRMWARE_ELF")]
+    firmware_elf: PathBuf,
+}
+
+fn spi_device_tx_rx_test(
+    opts: &Opts,
+    transport: &TransportWrapper,
+    tx_data: &[u8],
+    rx_data: &[u8],
+) -> Result<()> {
+    let uart = transport.uart("console")?;
+
+    /* Wait sync message. */
+    let _ = UartConsole::wait_for(&*uart, r"SYNC: TX_FIFO READY\r\n", opts.timeout)?;
+    let spi = transport.spi(&opts.spi)?;
+
+    let mut buf = vec![0u8; tx_data.len() - 1];
+    // Due to the limitations of hyper310, it does not support `Transfer::Both`, so we first sent
```

Comment:
`/sent/send`

Reply:
Thanks for spotting this, turns out that the comment was outdated, so I rephrased it.